pkgs/development/libraries/clutter/default.nix

```diff
@@ -1,32 +1,32 @@
 { stdenv, fetchurl, glib, pkgconfig, mesa, libX11, libXext, libXfixes
 , libXdamage, libXcomposite, libXi, cogl, pango, atk, json_glib, 
-gobjectIntrospection, gtk3
+gobjectIntrospection 
 }:
 
 let
-  ver_maj = "1.16";
-  ver_min = "2";
+  ver_maj = "1.26";
+  ver_min = "0";
 in
 stdenv.mkDerivation rec {
   name = "clutter-${ver_maj}.${ver_min}";
 
   src = fetchurl {
     url = "mirror://gnome/sources/clutter/${ver_maj}/${name}.tar.xz";
-    sha256 = "0hnz6fnrkc7ixrm2x83sxyha32p9896d7ilzhvxwfgzlh26fidqc";
+    sha256 = "01nfjd4k7j2n3agpx2d9ncff86nfsqv4n23465rb9zmk4iw4wlb7";
   };
 
   nativeBuildInputs = [ pkgconfig ];
   propagatedBuildInputs =
     [ libX11 mesa libXext libXfixes libXdamage libXcomposite libXi cogl pango
-      atk json_glib gobjectIntrospection gtk3
+      atk json_glib gobjectIntrospection
     ];
 
   configureFlags = [ "--enable-introspection" ]; # needed by muffin AFAIK
 
   #doCheck = true; # no tests possible without a display
 
   meta = {
-    description = "Library for creating fast, dynamic graphical user interfaces";
```

Comment:
Okay, I'll remove it. I didn't Touch the file btw, just moved old -> default.nix